src/common/pool_map.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2018 Intel Corporation.
+ * (C) Copyright 2016-2020 Intel Corporation.
```

Comment:
After landing my patch and merging, my githook changed all these copyrights because it detected changes...so probably just leave them because it means these files were actually changed in 2020